backend/app/categories/controller.py

```diff
@@ -19,22 +19,22 @@ async def get_category_route(category_id: str,
                              db: AsyncIOMotorClient = Depends(get_database)) -> ItemsInResponse:
     """Get the items under a given category"""
         
-    loader = None
-    if category_id == CategoryEnum.movies.value:
-        loader = Movie
-    if category_id == CategoryEnum.anime.value:
-        loader = Anime
-    if category_id == CategoryEnum.music.value:
-        loader = Music
-    if category_id == CategoryEnum.shows.value:
-        loader = Show
-    if category_id == CategoryEnum.books.value:
-        loader = Book
-    if category_id in CategoryEnum.__members__:
+    if category_id not in CategoryEnum.__members__:
+        return ItemsInResponse(success=False, error=[f"Invalid category {category_id}"])
+    else:
+        loader = None
+        if category_id == CategoryEnum.movies.value:
```

Comment:
you can directly do:
```python
if category_id == CategoryEnum.movies:
    pass
# same for other condition checks
```

Reply:
@kjain1810 you could have done this as well in the same commit ...

Reply:
This condition always returns false as CategoryEnum.movies does not return a string but an object of type <enum 'CategoryEnum'>. This is the reason I had to lookup and use the .value member.

Reply:
I have fixed the issue. The problem was the Enum class has to be declared as `class CategoryEnum(str, Enum)` instead of class CategoryEnum(Enum)` for it to work. Please change the same and commit!